tests/system_tests/appstart_systemtests.py

```diff
@@ -80,15 +88,23 @@ def _endpoint_test(self):
         self.assertEqual(res.status_code,
                          200,
                          '%s failed with error \"%s\"' %
-                         (handler.__name__, res.text))
+                         (endpoint, res.text))
     _endpoint_test.__name__ = 'test_%s_endpoint' % endpoint.strip('/')
     return _endpoint_test
 
 
 if __name__ == '__main__':
     logging.getLogger('appstart').setLevel(logging.INFO)
+
+    # Sync with urls in services_test_app.py
+    # Keeping handler as None for later on customizing of tests
+    urls = [('/datastore', None),
+        ('/logging', None),
+        ('/memcache', None)]
+
```

Comment:
Using None values for the handlers will break the error reporting (the assertEqual in make_endpoint_test() uses handler.__name__.

Are you actually running this as a script somewhere?  If so, I suggest changing make_endpoint_test() and just reporting `endpoint` instead of `handler.__name__`.